src/usr/local/www/status_openvpn.php

```diff
@@ -97,7 +97,7 @@ function(index,row) { $(row).fadeOut(1000); }
 ?>
 
 <div class="panel panel-default">
-		<div class="panel-heading"><h2 class="panel-title"><?=htmlspecialchars($server['name']);?> <?=gettext('Client Connections')?></h2></div>
+		<div class="panel-heading"><h2 class="panel-title"><?=htmlspecialchars($server['name']);?> <?=gettext('Client Connections') . ": "?>  <? if ($server['conns'][0]['common_name'] != '[error]'): echo sizeof($server['conns']); endif; ?></h2></div>
```

Comment:
you are closing php block and opening it again after a space `?> <?`. It's  better off not closing it and use a ternary operator to print what you need using the same construction inside `<?=`